README.md

```diff
@@ -0,0 +1,255 @@
+
+* To create an **superuser account**, use this command:
+``` python
+    $ python manage.py createsuperuser
+```
+For convenience, you can keep your normal user logged in on Chrome and your superuser logged in on Firefox (or similar), so that you can see how the site behaves for both kinds of users.
+
+### Type checks
+
+
+Running type checks with mypy:
+
+``` python
+    $ mypy codershq
+```
+
+### Test coverage
+
+
+To run the tests, check your test coverage, and generate an HTML coverage report:
+``` html
+    $ coverage run -m pytest
+    $ coverage html
+    $ open htmlcov/index.html
+```
+
+Running tests with py.test
+
+``` python
+    $ pytest
+```
+
+### Celery
+
+
+This app comes with Celery.
+
+To run a celery worker:
+
+``` shell
+    cd codershq
+    celery -A config.celery_app worker -l info
+```
+
+Please note: For Celery's import magic to work, it is important *where* the celery commands are run. If you are in the same folder with *manage.py*, you should be right.
+
+
+### Email Server
+
+
+In development, it is often nice to be able to see emails that are being sent from your application. For that reason local SMTP server [MailHog](https://github.com/mailhog/MailHog) with a web interface is available as docker container.
+
+Container mailhog will start automatically when you will run all docker containers.
+Please check `cookiecutter-django Docker documentation`_ for more details how to start all containers.
+
+With MailHog running, to view messages that are sent by your application, open your browser and go to ``http://127.0.0.1:8025``
+
+
+
+
+
+### :rocket: Deployment
+
+
+The following details how to deploy this application.
+
+
+### Heroku 
+
+See detailed [cookiecutter-django Heroku documentation](http://cookiecutter-django.readthedocs.io/en/latest/deployment-on-heroku.html)
+
+
+
+
+### :whale: Docker
+
+
+See detailed [cookiecutter-django Docker documentation](http://cookiecutter-django.readthedocs.io/en/latest/deployment-with-docker.html).
+
+
+### Custom Bootstrap Compilation
+
+
+The generated CSS is set up with automatic Bootstrap recompilation with variables of your choice.
+Bootstrap v4 is installed using npm and customised by tweaking your variables in ``static/sass/custom_bootstrap_vars``.
+
+You can find a list of available variables [in the bootstrap source](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss), or get explanations on them in the [Bootstrap docs](https://getbootstrap.com/docs/4.1/getting-started/theming/).
+
+
+
+<p><a href="https://codershq-challenge.slack.com"><h2>Slack</h2><img width="80" src="https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white">
+</a></p> 
+
+
+This project has an associated slack page that is used when a new challenge is created and discussions which relates to challenges in general. The slack page is located at  [codershq-challenge.slack.com](https://codershq-challenge.slack.com).
```

Comment:
We don't use bootstrap since we moved to tailwind css and slack is not supported anymore since we have discord